examples/cluster-configuration/services-configuration.yaml

```diff
@@ -132,9 +132,3 @@ prometheus:
   #    smtp_from: foo_bar@gmail.com
   #    smtp_auth_username: user@gmail.com
   #    smtp_auth_password: gmail_password
-
-
-pylon:
-  # port of pylon
-  port: 80
-
```

Comment:
should also change `deployment/quick-start/services-configuration.yaml.template`. When change it, comment out all pylon default value, and leave a comment, so quick start user can simple uncomment value to change.

Reply:
Sorry for a bad push, please refresh the PR